README.md

```diff
@@ -217,6 +217,8 @@ Sending data to Datadog US (default) and configuring a few checks.
   roles:
     - { role: Datadog.datadog, become: yes }
   vars:
+    system_probe_config:
```

Comment:
Please move this section after the `datadog_config` one. We try to have the most used option first.
Moving it line 274 or 279 would be better.